pkg/ec2metadata/ec2metadata.go

```diff
@@ -322,12 +321,12 @@ func (e *Service) GetNodeMetadata() NodeMetadata {
 	var metadata NodeMetadata
 	identityDoc, err := e.GetMetadataInfo(IdentityDocPath)
 	if err != nil {
-		log.Log().Err(err).Msg("Unable to fetch metadata from IMDS")
+		log.Err(err).Msg("Unable to fetch metadata from IMDS")
 		return metadata
 	}
 	err = json.NewDecoder(strings.NewReader(identityDoc)).Decode(&metadata)
 	if err != nil {
-		log.Log().Msg("Unable to fetch instance identity document from ec2 metadata")
+		log.Warn().Msg("Unable to fetch instance identity document from ec2 metadata")
```

Comment:
I don't think we can make a strict rule like that. We have a fallback for this error, so I think it can stay a Warn.

Reply:
Agreed the rule can't be that strict, but if having a fallback doesn't classify as error, then what about:

`ERR Unable to retrieve an IMDSv2 token, continuing with IMDSv1`

v1 is the fallback in this case? Are there any formal docs for log-levels similar to semantic versioning or something? Agreed we need some kinda consistency

Reply:
yeah I changed that one to WARN

Reply:
I don't think there's anything formal for log levels... I think we probably log way too much in NTH, so maybe we should consider what actually makes sense to log so that someone can view the logs and be confident things are working and have enough information to debug certain error scenarios.